### ISSUE TITLE
Agree field description translation not working when field contains special character

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#216]
**Description**
If I insert French, for example, in the description of an agree field which contains a character such as `é` the translation via the static formie.php translation file does not work.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.27",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.2.28](https://github.com/verbb/formie/releases/tag/1.2.28)
